pkg/config/engine/containerd/config_v1.go

```diff
@@ -157,3 +173,15 @@ func (c *ConfigV1) Set(key string, value interface{}) {
 func (c ConfigV1) Save(path string) (int64, error) {
 	return (Config)(c).Save(path)
 }
+
+func (c *ConfigV1) GetRuntimeConfig(name string) (engine.RuntimeConfig, error) {
+	if c == nil || c.Tree == nil {
+		return nil, fmt.Errorf("config is nil")
+	}
+	config := *c.Tree
```

Comment:
I don't think we strictly speaking need to grab the value here since we're not updating the tree.

Reply:
Sure